src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5214,6 +5214,7 @@ namespace OutputReportTabular {
         static Array1D<Real64> radiantCool;
         static Array1D<Real64> ATUHeat;
         static Array1D<Real64> ATUCool;
+        static Real64 ZoneEqHeatorCool(0.0);
```

Comment:
Nit-picking, but we don't want to declare this here, and we don't want it to be static. Just declare it in the loop as noted below.